contracts/test/_metastrategies-fixtures.js

```diff
@@ -0,0 +1,221 @@
+
+  // Tilt to 3CRV by half a million
+  const exchangeSign = "exchange(int128,int128,uint256,uint256)";
+  await ousdMetaPool.connect(domen)[exchangeSign](0, 1, amount, 0);
+
+  await vault.connect(domen).allocate();
+  await vault.connect(domen).rebase();
+}
+
+async function get3CRVLiquidity(fixture, crv3Amount) {
+  const { threepoolSwap } = fixture;
+  const vPrice = await threepoolSwap.get_virtual_price();
+  return crv3Amount.div(vPrice).mul(ousdUnits("1"));
+}
+
+async function addLiquidity(fixture, ousdAmount, crv3Liquidity, receiver) {
+  const { domen, ousdMetaPool } = fixture;
+
+  receiver = receiver ? receiver.getAddress() : domen.getAddress();
+
+  const addSignature = "add_liquidity(uint256[2],uint256,address)";
+  const addMethod = await ousdMetaPool.connect(domen)[addSignature];
+  addMethod([ousdAmount, crv3Liquidity], 0, receiver);
+}
+
+async function removeAllLiquidity(fixture, user) {
+  const { ousdMetaPool } = fixture;
+
+  const address = user.getAddress();
+  const removeSign = "remove_liquidity(uint256,uint256[2])";
+  const removeMethod = await ousdMetaPool.connect(user)[removeSign];
+  removeMethod(ousdMetaPool.balanceOf(address), ["0", "0"]);
+}
+
+async function removeLiquidityImbalanced(fixture, user, crv3Liquidity) {
+  const { ousdMetaPool } = fixture;
+
+  const address = user.getAddress();
+  const removeImbalanceSign = "remove_liquidity_imbalance(uint256[2],uint256)";
+  await ousdMetaPool.connect(user)[removeImbalanceSign](
+    ["0", crv3Liquidity],
+    crv3Liquidity.mul("12").div("10") // 20% of tolerance
+  );
+  const removeOneCoinSign = "remove_liquidity_one_coin(uint256,int128,uint256)";
+  await ousdMetaPool.connect(user)[removeOneCoinSign](
+    await ousdMetaPool.balanceOf(address),
+    0, // pool token ID. 0: OUSD, 1: 3CRV
+    0 // Burn none
+  );
+}
+
+async function withLiquidityOnBalancedPool() {
+  const fixture = await loadFixture(withDefaultStrategiesSet);
+  // 10k OUSD
+  const ousdAmount = ousdUnits("10000");
+  // $10k of 3CRV
+  const crv3Liquidity = await get3CRVLiquidity(fixture, ousdAmount);
+
+  // Daniel has added balanced liquidity
+  await addLiquidity(fixture, ousdAmount, crv3Liquidity, fixture.daniel);
+
+  // Franck has added imbalanced liquidity
+  await addLiquidity(fixture, ousdAmount, crv3Liquidity.div(2), fixture.franck);
+
+  // Balance metapool
+  await balanceMetaPool(fixture);
+
+  return fixture;
+}
+
+async function withLiquidityOnOUSDTitledPool() {
+  const fixture = await loadFixture(withBalancedMetaPool);
+  // 10k OUSD
+  const ousdAmount = ousdUnits("10000");
+  // $10k of 3CRV
+  const crv3Liquidity = await get3CRVLiquidity(fixture, ousdAmount);
+
+  // Daniel has added balanced liquidity
+  await addLiquidity(fixture, ousdAmount, crv3Liquidity, fixture.daniel);
+
+  // Franck has added imbalanced liquidity
+  await addLiquidity(fixture, ousdAmount, crv3Liquidity.div(2), fixture.franck);
+
+  // Balance metapool
+  await balanceMetaPool(fixture);
```

Comment:
My reasoning was not adding `loadFixture` inside those methods is because the tiltTo3Crv function is also used by `withCRV3TitledMetapool` where we load the balanced metapool. But yeah, I could rewrite this